service/src/chain_spec.rs

```diff
@@ -151,25 +175,33 @@ pub fn get_account_id_from_seed(seed: &str) -> AccountId {
 		.public()
 }
 
+/// Helper function to generate BabeId from seed
+pub fn get_babe_id_from_seed(seed: &str) -> BabeId {
+	sr25519::Pair::from_string(&format!("//{}", seed), None)
+		.expect("static values are valid; qed")
+		.public()
+}
+
 /// Helper function to generate SessionKey from seed
-pub fn get_session_key_from_seed(seed: &str) -> SessionKey {
+pub fn get_grandpa_id_from_seed(seed: &str) -> SessionKey {
```

Comment:
if it is returning `SessionKey` then it's probably not `grandpa_id`

Reply:
This file _might still work_ but __for sure__ has a confused terminology now that the new crypto-app-stuff is in. Someone should have a look at it later.